--- gccxml-0.6.0/GCC_XML/KWSys/SystemTools.cxx.orig	2004-02-04 23:50:19.000000000 +0100
+++ gccxml-0.6.0/GCC_XML/KWSys/SystemTools.cxx	2015-10-06 22:07:00.491875951 +0200
@@ -26,6 +26,7 @@
 #include <sys/stat.h>
 #include <ctype.h>
 #include <errno.h>
+#include <string.h>
 #include <time.h>
 
 // support for realpath call
--- gccxml-0.6.0/GCC_XML/GXFront/gxConfiguration.cxx.orig	2004-02-04 23:50:19.000000000 +0100
+++ gccxml-0.6.0/GCC_XML/GXFront/gxConfiguration.cxx	2015-10-06 22:12:09.615196312 +0200
@@ -16,6 +16,8 @@
 =========================================================================*/
 #include "gxConfiguration.h"
 
+#include <string.h>
+
 //----------------------------------------------------------------------------
 const char* gxConfigurationVc6Registry =
 "HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\"
